manifests/proxydhcp.pp

```diff
@@ -53,6 +53,8 @@
 
   if $foreman_proxy::dhcp_manage_acls {
 
+    Concat<| title == "${dhcp::dhcp_dir}/dhcpd.conf" |> { mode => '0640' }
```

Comment:
Looking back at this, it may have broken things - it doesn't set the group or permissions for Foreman Proxy to read this file.